### PR TITLE
Add twitter badge to readme

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

**Comment by dwelle** [comment]
Doesn't look like twitter. Let's use the default: https://img.shields.io/twitter/follow/excalidraw?style=social

**Comment by lipis** [comment]
That's also not the official, but I took it from Prettier.. https://github.com/prettier/prettier#opinionated-code-formatter

**Comment by dwelle** [comment]
I'd not sacrifice legibility for style. If nothing else, let's add an icon to it:

https://img.shields.io/twitter/follow/excalidraw.svg?label=follow+excalidraw&style=flat-square&logo=twitter

otherwise scanning the readme I'd never notice there's a twitter badge at all.

But whatever, let's not get hung up on this.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-    <a href="https://excalidraw.com">
-        <img src="./public/og-image.png" alt="Excalidraw logo: Sketch handrawn like diagrams." />
-    </a>
-    <h3>Excalidraw is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them.</h3>
+  <a href="https://excalidraw.com">
+    <img src="./public/og-image.png" alt="Excalidraw logo: Sketch handrawn like diagrams." />
+  </a>
+  <h3>Excalidraw is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feel to them.</h3>
+  <a href="https://twitter.com/Excalidraw">
+    <img alt="Follow Excalidraw on Twitter" src="https://img.shields.io/twitter/follow/excalidraw.svg?label=follow+excalidraw&style=flat-square">
```
Comment (lipis) [inline README.md]:
```suggestion
    <img alt="Follow Excalidraw on Twitter" src="https://img.shields.io/twitter/follow/excalidraw.svg?label=follow+excalidraw&style=flat-square&logo=twitter">
```

**Comment by lipis** [comment]
Updated https://github.com/excalidraw/excalidraw/blob/twitter/README.md

**Comment by dwelle** [comment]
Alright, thx. Merging...

**Pull request merged**